repose-aggregator/components/services/rate-limiting-service/src/main/java/org/openrepose/core/services/ratelimit/LimitKey.java

```diff
@@ -45,12 +40,12 @@ public static String getLimitKey(String limitGroup, String limitId, Matcher uriM
         cacheIdBuffer.append(String.valueOf(limitGroup.hashCode()));
 
         // All cacheId's contain the unique limit Id
-        cacheIdBuffer.append(":" + String.valueOf(limitId.hashCode()));
+        cacheIdBuffer.append(":").append(String.valueOf(limitId.hashCode()));
```

Comment:
no it was @dmnjohns, suspect he just added what was needed and wasn't paying attention to the overall context.